src/bindings/python/src/openvino/frontend/pytorch/patch_model.py

```diff
@@ -84,11 +84,12 @@ def __make_16bit_traceable(model: torch.nn.Module,
      - Replace known list of modules with ModuleExtension.
      - Convert other modules with weights to FP32.
```

Comment:
need to change a name of routine `__make_16bit_traceable` -> `__make_compressed_layers_traceable`